lib/utils.js

```diff
@@ -117,21 +117,10 @@ module.exports.processMethod = async function (msg, cfg) {
     headers: {
       Authorization: "Bearer " + token,
     },
-    body: {
-      name: msg.body.data.name,
-      first_name: msg.body.data.first_name,
-      last_name: msg.body.data.name,
-      phone: msg.body.data.phone,
-      email: msg.body.data.email,
-    },
-
-    data: {
-      name: msg.body.data.name,
-      first_name: msg.body.data.first_name,
-      last_name: msg.body.data.name,
-      phone: msg.body.data.phone,
-      email: msg.body.data.email,
-    },
+    data: msg.data
```

Comment:
There are other locations in this file where requestOptions.body is called. Removing the key completely will break them. All of these would need to be updated to `requestOptions.data`

Reply:
But the body is used by axios?